src/Language/PureScript/Bridge/Printer.hs

```diff
@@ -58,21 +59,36 @@ moduleToText m = T.unlines $
   "-- File auto generated by purescript-bridge! --"
   : "module " <> psModuleName m <> " where\n"
   : map importLineToText (Map.elems (psImportLines m))
-  ++ [ "\nimport Data.Generic (class Generic)\n\n" ]
+  ++ [ ""
+     , "import Prelude"
+     , "import Data.Generic (class Generic)"
+     , "import Data.Maybe"
+     , "import Data.Lens"
+     , ""
+     ]
```

Comment:
ideally we would add all those imports to the generated module psImportLines, this way also duplicate imports would be avoided. I can't remember why I did not do this back then, but now that there are more imports lines added, it seems to be a good idea.